test/instrumentation/external/nocode/nocode.test.ts

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as path from 'path';
+import * as fs from 'fs';
+import { strict as assert } from 'assert';
+import { after, before, beforeEach, describe, it } from 'node:test';
+import { NoCodeInstrumentation } from '../../../../src/instrumentations/external/nocode';
+import { getTestSpans, setInstrumentation, provider, exporter } from '../setup';
+
+const configPath = './test/instrumentation/external/nocode/nocode.config.json';
+const absolutePathToUtils = process.env.GITHUB_WORKSPACE
+  ? path.join(
+      process.env.GITHUB_WORKSPACE,
+      'test/instrumentation/external/nocode/sample-utils.ts'
+    )
+  : path.resolve(__dirname, 'sample-utils.ts');
```

Comment:
Don't these two end up in the same path anyway? (github actions check seems redundant)